docs/src/references/child-devices.md

```diff
@@ -0,0 +1,120 @@
+  a _logical child-device_ or an _external-device_ that makes use of thin-edge's child-device API to be represented as individual device in the cloud
+- **child-device twin**:
+  the entity in the cloud, that represents the child-device; in case of C8Y that is a managed object
+- **child-id**:
+  a string uniquely referencing the _external-device_ and the corresponding _child-device twin_
+- **child-device agent**:
+  that piece of software that makes the glue between the _external-device_ and thin-edge APIs; could be located on the _external-device_ or running as process on the _thin-edge device_
+
+## Principles
+
+Child-device support basically focuses on three aspects:
+   1) provision the child-device, i.e. making thin-edge and the cloud aware of the child-device (external or logical) 
+   2) thin-egde APIs provided on network, to enable external APIs to access them
+   3) associate consumed/provided data of a child-device (external or logical) to it's child-device twin in the cloud
+
+### (1) Child-Device Provisioning
+
+A newly attached child-device (external or logical) must be once provisioned. The provisioning phase of a child-device includes:
+  1) creating cloud's child-device twin
+  2) declaring all supported capabilities of the child-device to its child device twin
+     (i.E. declaring all _supported operations_ and all _types_ per supported operation, as _config types_, _log types_, ...)
+     (i.e. declaring all operations supported by the device like configuration management, software management etc and further metadata per supported operation, like _config types_, _log types_, _software list_ etc)
+
+There are two option to create the cloud's child-device twin and declaring all supported capabilities. 
+
+On the one hand a cloud's _child-device twin_ can be created upfront (e.g. with some customer-specific cloud-site backend), and also all capabilities can be declared that way to the new twin. Then the child-device on the device site (external or logical) relies on that existing twin.
+
+On the other hand a child-device (external or logical) can announce it's capabilities to thin-edge to provoke thin-edge creating the cloud's child-device twin and declaring all capabilities on that child-device twin. 
+
+##### Announcing child-device capabilities by the child-device it-self
+
+To announce it's capabilities the child device (external or logical) sends an MQTT message to `tedge/meta/<childid>`. The MQTT message is as below:
+
+```json
+{
+   "device-name": "<optional name of that child-device>",
+   "device-type": "<optional type of that child-device>",
+   "<capability 1>": <capability specific JSON object>,
+   "<capability 2>": <capability specific JSON object>,
+   // [...]
+}
+```
+
+Thereby the fields are as below:
+   * `device-name` is an optional human readble device-name, visible in the cloud. If the field is not contained in the JSON message the `childid` of the topic structure is used as device-name.
```

Comment:
Does it mean that the `device-name` declared here can be different than the one that is present in the `topic`?

Reply:
Yes. The device-name is a human readable string. Instead the MQTT topic structure contains the unique child-id.

To speak in C8Y terms:
* the child-id is the external-id, that is used to address the device
* the device name is just a string, e.g. visiable in the UI

See also https://cumulocity.com/guides/reference/smartrest-two/#child-device-creation-101